en/PlanView/PlanView.md

```diff
@@ -1,47 +1,48 @@
 
-The image above shows a simple mission which starts with a takeoff, flies through two waypoints and then lands.
+The image above shows a simple mission which starts with a takeoff at the home position (H), flies through three waypoints and then lands on the last waypoint (i.e. waypoint 3).
 
 The steps to creating a missions are:
 
-1. Change to Plan View
-2. Add commands to the mission and edit as needed
-3. Send the mission to the vehicle
-4. Change to Fly View and fly your mission
+1. Change to *Plan View*.
+2. Add waypoints or commands to the mission and edit as needed.
+3. Upload the mission to the vehicle.
+4. Change to *Fly View* and fly your mission.
 
 ## Plan Tools
-On the left edge of the screen you will see the Plan Tools. The order of tools from top to bottom is:
+On the left edge of the screen you will see the Plan Tools as marked in red in the image above. 
 
-* Add Commands
-* [Survey](Survey.md)
-* Sync
-* Center map
-* Map Type
-* Zoom In/Out
+ > **Note** **Center map**, **Zoom In**, **Zoom Out** are just map visualisation tools that help user view the *Plan view* map clearly while planning missions (they don't affect the mission commands sent to the vehicle in any way).
 
-### Add Commands
-Click to activate the Add Commands tool. While active, clicking in the map will add new mission commands at the clicked location. The tool will stay active until you click it again.
+
+### Add Waypoints
+Click to activate the **Add Waypoint** tool. While active, clicking in the map will add new mission waypoint at the clicked location. The tool will stay active until you click it again. Once you have added a waypoint, you can select it by clicking on it and then drag it around to change its position.
+
+### Actual vs Planned Home Position
+Home position is the position the vehicle will return to and land on when in Return (to Launch/Home) mode. The home position shown in *Plan View* is merely a virtual home position a.k.a "Planned Home Position" which helps you simulate the vehicle's home position while planning a mission. Hence, you should place it where you roughly plan to start (takeoff) the vehicle from. This allows QGC to estimate mission times and to draw waypoint lines correctly to the home position. When you go out on the field and start the mission, the autopilot then replaces the mission's planned home position with the actual takeoff position as its home position. This actual home position can then be seen in the *Fly View*.
+
+<img src="../../images/plan/MissionSettingsPlannedHomePositionSection.jpg" style="width: 300px;"/>
 
 ### Sync
 The Sync tools allows you to move Missions back and forth to your Vehicle or a file. *Before you fly a mission you must be sure to send your Mission to your vehicle.* The tool will change to have an "!" within it to indicate that you have changes to your Mission which you have not sent to your vehicle. 
```

Comment:
"which" should often actually be "that" (I'm choosing a case here that you did not edit). This is a very picky point, but "that" is used where the following text is essential to the sentence, and "which" is used where it is some sort of additional information that you can cut off without losing the sentence. If you use "which" usually you will precede it with a comma. 

So here, **that** would be better.

Below we have a case where which is correct, but which would be better if the "which" was preceded by a comma :-)
" Click on a mission command to show its editor which allows you to specify the values for the command. " 

This is very FYI only - change if you feel inclined. People will know what you mean!